cmake/FindSEACASExodiff.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2016 Los Alamos Natl. Lab
```

Comment:
Ah! I understand that the trilinos-provided Find*cmake stuff is unavailable until it is found (as chicken and egg) so we need this. This is pretty stupid, but I don't see a better way of doing this, unless somehow concocting it from the trilinos source under tpl/src/trilinos. I just looked and it does not seem too straightforward to extract some kind of FindSEACASExodus.cmake and FindSEACASExodiff.cmake from it.

I guess adding these two files remains the most maintainable solution for now...